src/scrabble.js

```diff
@@ -1,6 +1,305 @@
+  doubleScore () {
+    const doubleScore = this.doubleScoreCurlyBrackets().map(letter => scoreObject[letter.toUpperCase()]).reduce((x, y) => x + y, 0)
+    console.log(doubleScore)
+    return doubleScore * 2
+  }
 
+  // triple score
+  tripleScore () {
+    const tripleScore = this.tripleScoreSquareBrackets().map(letter => scoreObject[letter.toUpperCase()]).reduce((x, y) => x + y, 0)
+    console.log(tripleScore)
+    return tripleScore * 3
+  }
+
+  scoresTotaled () {
+    this.totalScore = singleScore + doubleScore + tripleScore
+    console.log('.............', this.totalScore)
+    return this.totalScore
+  }
+}
+// -------------------------------------24-24-8-8-24-1-1-1-1-3-4-1
+const doubleScoreWord = new Scrabble('[X][X]{Y}{Y}[X]aaaapwu')
+const final = new Scrabble('xx')
+console.log('removeDoubleScoredLetters', final.removeDoubleScoredLetters())
+console.log('removeTripleScoredLetters', final.removeTripleScoredLetters())
+const wordArray = doubleScoreWord.wordArray()
+console.log('wordArray', wordArray)
+const scoredLettersDouble = doubleScoreWord.doubleScoreCurlyBrackets()
+const scoredLettersTriple = doubleScoreWord.tripleScoreSquareBrackets()
+console.log('scoredLettersDouble', scoredLettersDouble)
+console.log('scoredLettersTriple', scoredLettersTriple)
+
+// remove brackets from array
+
+const bracketsToRemove = ['[', ']', '{', '}']
+const arrayCleanOfBrackets = wordArray.filter(item => {
+  return bracketsToRemove.indexOf(item) === -1
+})
+console.log('arrayCleanOfBrackets.', arrayCleanOfBrackets)
+
+// remove double score letters from original array
+
+const arrayCleanOfDouble = arrayCleanOfBrackets.filter(item => {
+  return scoredLettersDouble.indexOf(item) === -1
+})
+console.log('arrayCleanOfDouble....', arrayCleanOfDouble)
+
+// remove triple score letters
+
+const arrayCleanDoubleTriple = arrayCleanOfDouble.filter(item => {
+  return scoredLettersTriple.indexOf(item) === -1
+})
+console.log('arrayCleanDoubleTriple', arrayCleanDoubleTriple)
+
+const scoreObject = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10
 }
+// single score
+
+const singleScore = arrayCleanDoubleTriple
+  .map(letter => scoreObject[letter.toUpperCase()])
+  .reduce((x, y) => x + y, 0)
+console.log('single score....', singleScore)
+
+// double score
+
+const doubleScore = scoredLettersDouble
+  .map(letter => scoreObject[letter.toUpperCase()])
+  .reduce((x, y) => x + y, 0) * 2
+
+console.log('double score....', doubleScore)
+
+// triple score
+
+const tripleScore = scoredLettersTriple
+  .map(letter => scoreObject[letter.toUpperCase()])
+  .reduce((x, y) => x + y, 0) * 3
+
+console.log('triple score....', tripleScore)
+
+const finalScore = singleScore + doubleScore + tripleScore
+console.log('final score....', finalScore)
 
 module.exports = Scrabble
```

Comment:
There is a lot to go through here. Although I think you may have gone down the wrong path to efficiently solve the scrabble challenge, you have demonstrated a good understanding of how to structure a very complicated class and make the methods work together. All the time you have spent researching different ways to manipulate data will help in future more complex projects